Frontend/ui-driver/src/Flow.purs

```diff
@@ -1058,6 +1058,9 @@ homeScreenFlow = do
   let (Vehicle linkedVehicle) = (fromMaybe dummyVehicleObject getDriverInfoResp.linkedVehicle)
   setValueToLocalStore USER_NAME getDriverInfoResp.firstName
   _ <- pure $ spy "response for mode zxc" getDriverInfoResp
+  if (isJust getDriverInfoResp.numberOfRides) then do
+    setValueToLocalStore HAS_TAKEN_FIRST_RIDE if ((fromMaybe 0 getDriverInfoResp.numberOfRides) == 0) then "true" else "false"
```

Comment:
instead of writing :-
`if ((fromMaybe 0 getDriverInfoResp.numberOfRides) == 0) then "true" else "false"`
we can just write :-
show $ (fromMaybe 0 getDriverInfoResp.numberOfRides) == 0